.env.example

```
@@ -1,65 +1,32 @@
-REDIS_PORT=6379
 
-MAIL_MAILER=log
-MAIL_SCHEME=null
-MAIL_HOST=127.0.0.1
-MAIL_PORT=2525
-MAIL_USERNAME=null
-MAIL_PASSWORD=null
-MAIL_FROM_ADDRESS="hello@example.com"
-MAIL_FROM_NAME="${APP_NAME}"
+ADMIN_USERNAME=admin
+ADMIN_PASSWORD=password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ADMIN_PASSWORD key should go before the ADMIN_USERNAME key

(UnorderedKey)